### PR TITLE
Add many things

### LINKED ISSUE #5: Update required
In follow up to https://github.com/sindresorhus/awesome/issues/1755

This awesome list is *seriously* outdated. 
>> The awesome repo for Minecraft under gaming is seriously outdated

We do not even have sections mentioning the existence of resource packs and data packs.
A serious revamp is required.

It would be nice to understand the availability and thoughts of the original maintainers on this.

### LINKED ISSUE #7: Add server software section
Useful for people trying to find suitable software.

For Java, I'd recommend [Paper](https://papermc.io/) but I don't know about Bedrock.

### PR DESCRIPTION
**Pull request by ThexXTURBOXx**
This adds many things in many sections to keep this a little bit more up-to-date :)

Fixes #5 
Fixes #7

### CODE REVIEW

**Comment by g-plane** [comment]
Huge thanks! I will review ASAP.

**Comment by ThexXTURBOXx** [comment]
No problem :) Sorry for doing so many commits! I wanted to check out StackEdit... Didn't know it would do that

**Comment by g-plane** [comment]
That will be better. Thank you!

**Comment by ThexXTURBOXx** [comment]
Done. Feel free to review and/or merge now! :)

File: readme.md [hunk 1]
```diff
@@ -147,6 +168,24 @@
...
+- [Soartex](https://soartex.net/) - Smooth resource pack with mod support.
+- [Faithful](https://faithful.team/) - Smooth textures for a furbished default look.
+
+## Shader Packs
+
+- [Continuum](https://continuum.graphics/) - A realistic shader pack.
+- [Chocapic13](https://www.curseforge.com/minecraft/customization/chocapic13-shaders) - A realistic and configurable shader pack.
+- [Sonic Ether](https://www.sonicether.com/) - Pathracing and other realistic shader packs.
+- [Sildur](https://sildurs-shaders.github.io/) - Shader packs even implementing Minecraft 1.16's built-in API.
+- [Potato](https://rre36.com/potato-shader) - A very lightweight shader pack for... well... potato PCs.
```
Comment (g-plane) [inline readme.md:186]: What's the `...`?

File: readme.md [hunk 2]
```diff
@@ -147,6 +168,24 @@
...
+- [ESBE 2G](https://mcbeeringi.github.io/esbe-2g/) - A shader pack for Bedrock Edition
```
Comment (g-plane) [inline readme.md:187]:
```suggestion
- [ESBE 2G](https://mcbeeringi.github.io/esbe-2g/) - A shader pack for Bedrock Edition.
```

File: readme.md [hunk 3]
```diff
@@ -155,6 +194,20 @@
...
+- [Spigot](https://www.spigotmc.org/) - CraftBukkit's successor providing support for the Bukkit API.
+- [Sponge](https://www.spongepowered.org/) - Minecraft Server providing support for the Sponge API and is extensible to the Forge API and Bukkit API.
+- [Mohist](https://mohistmc.com/) - A server implementing the Forge and Bukkit API.
+- [Thermos](https://cyberdynecc.github.io/Thermos/install) - A server implementing the Forge and Bukkit API for 1.7.10.
+- [KCauldron](https://sourceforge.net/projects/kcauldron/files/) - A server implementing the Forge and Bukkit API for 1.7.10.
+- [Cauldron](https://sourceforge.net/projects/cauldron-unofficial/files/) - Formerly known as MCPC+. The original server implementing the Forge and Bukkit API.
+- [PowerNukkit](https://powernukkit.org/) - A powerful fork of Nukkit.
+- [Nukkit/NukkitX](https://cloudburstmc.org/articles/) - A Bedrock Edition Server providing support for plugins supporting the Nukkit API.
+- [PocketMine](https://www.pocketmine.net/?lang=de) - A Bedrock Edition Server providing support for basic plugins.
```
Comment (g-plane) [inline readme.md:209]:
```suggestion
- [PocketMine](https://www.pocketmine.net/) - A Bedrock Edition Server providing support for basic plugins.
```

File: readme.md (hunk 1 above)
Comment (ThexXTURBOXx) [reply readme.md:186]: It's a little joke if that's okay. I figured, one could make this list a little bit more interesting by adding stuff like this (It's the only one right now)

Comment (g-plane) [reply readme.md:186]: Joke is okay, but I'm afraid whether others can understand this description or not.

Comment (ThexXTURBOXx) [reply readme.md]: Is that better?

Comment (g-plane) [reply readme.md]: :+1:

**Review by g-plane — APPROVED** [review]
Thank you!

**Pull request merged**